mlflow/server/js/src/experiment-tracking/components/ExperimentListView.js

```diff
@@ -173,55 +264,66 @@ export class ExperimentListView extends Component {
+          <PlusCircleBorderIcon
+            onClick={this.handleCreateExperiment}
+            css={{
+              fontSize: '24px',
+              marginLeft: 'auto',
             }}
-          >
-            <Typography.Title level={2} style={{ margin: 0 }}>
-              Experiments
-            </Typography.Title>
-            <PlusCircleBorderIcon
-              onClick={this.handleCreateExperiment}
-              css={{
-                fontSize: '24px',
-                marginLeft: 'auto',
-              }}
-              title='New Experiment'
-              data-test-id='create-experiment-button'
-            />
-            <CaretDownSquareIcon
-              onClick={() => this.setState({ hidden: true })}
-              rotate={90}
-              css={{ fontSize: '24px' }}
-              title='Hide experiment list'
-            />
-          </div>
+            title='New Experiment'
+            data-test-id='create-experiment-button'
+          />
+          <CaretDownSquareIcon
+            onClick={() => this.setState({ hidden: true })}
+            rotate={90}
+            css={{ fontSize: '24px' }}
+            title='Hide experiment list'
+          />
+        </div>
+        <div css={classNames.experimentSearchContainer}>
           <Input
             placeholder='Search Experiments'
             aria-label='search experiments'
             value={searchInput}
             onChange={this.handleSearchInputChange}
+            onPressEnter={this.handleLoadMore}
             data-test-id='search-experiment-input'
+            css={classNames.experimentSearchInput}
           />
-          <div css={classNames.experimentListContainer}>
-            <Tree
-              treeData={treeData}
-              dangerouslySetAntdProps={{
-                selectable: true,
-                checkable: true,
-                multiple: true,
-                selectedKeys: activeExperimentIds,
-                checkedKeys: activeExperimentIds,
-                onCheck: this.handleCheck,
-                titleRender: this.renderListItem,
-              }}
-            />
-          </div>
+          <SearchIcon
+            onClick={this.handleLoadMore}
+            title='Search/load more experiments from database'
+            css={classNames.experimentSearchIcon}
+          />
+        </div>
+        <div>
+          <AutoSizer>
+            {({ width, height }) => (
+              <InfiniteLoader
+                isRowLoaded={({ index }) => !!experiments[index]}
+                loadMoreRows={this.onScroll}
+                rowCount={9999999} // arbitrarily high value
+              >
+                {({ onRowsRendered, registerChild }) => (
+                  <VList
+                    rowRenderer={this.renderListItem}
+                    data={this.state.checkedKeys}
+                    onRowsRendered={onRowsRendered}
+                    ref={this.bindListRef}
+                    rowHeight={30}
```

Comment:
I think that we can bring back previous 32px height
```suggestion
                    rowHeight={32}
```